File: backend/src/main/java/Manager/Restaurant/mai/dto/RegisterRequest.java
package Manager.Restaurant.mai.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RegisterRequest {
    private String userSlug;
    private String userName;
    private String userEmail;
    private String userPassword;
    private String userSalt;
    private String userGender;
    private String userPhone;
    private String userAvatar;
    private String userStatus;
    private LocalDateTime userDob;
    private Long roleId;
}

File: frontend/36Food/app/src/main/java/com/example/a36food/data/dto/RegisterRequest.kt
package com.example.a36food.data.dto

data class RegisterRequest(
    val userEmail: String,
    val userName: String,
    val userPassword: String,
    val userPhone: String,
    val userGender: String?,
    val userDob: String?,
    val userSlug: String,
    val roleId: Int,
    val userStatus: String = "ACTIVE",
    val userAvatar: String? = null,
    val userSalt: String? = null
)